pkgs/servers/http/router/default.nix

```diff
@@ -35,15 +35,15 @@ rustPlatform.buildRustPackage rec {
   # To avoid this we pre-download the file and export it via RUSTY_V8_ARCHIVE
   RUSTY_V8_ARCHIVE = callPackage ./librusty_v8.nix { };
 
+  doCheck = false;
```

Comment:
a comment could be nice about why all tests are disabled

Reply:
my bad. It's removed.